docs/source/examples/ggi_summer_school_unsolved.md

```diff
@@ -0,0 +1,326 @@
+    # TODO: pass the "folding_function" and the "factory" as arguments. 
+    # See docs: https://mitiq.readthedocs.io/en/stable/apidoc.html#module-mitiq.zne.zne
+)
+factory.plot_fit()
+print(f"The error mitigated expectation value is <A> = {zne_value}")
+```
++++
+## What happens behind the scenes? A low-level application of ZNE
+
+In Mitiq one can indirectly amplify noise by intentionally increasing the depth of the circuit in different ways.
+
+For example, the function `zne.scaling.fold_gates_at_random()` applies transformation $G \rightarrow G G^\dagger G$ to each gate of the circuit (or to a random subset of gates).
+
++++
+### STEP 1: Noise-scaled expectation values are evaluated via gate-level "unitary folding" transformations
+
+```{code-cell} ipython3
+locally_folded_circuit = # apply fold_gates_at_random() to "circuit" with scale factor of 3.
+# Link to docs: https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.zne.scaling.folding.fold_gates_at_random
+
+print("Locally folded GHZ circuit:")
+print(locally_folded_circuit)
+```
+
+**Note:** To get a simple visualization, we did't apply the preliminary circuit transpilation that we used in the previous section.
+
+Alternatively, the function `zne.scaling.fold_global()` applies the transformation $U \rightarrow U U^\dagger U$ to the full circuit.
+
+```{code-cell} ipython3
+globally_folded_circuit = # apply fold_global() to "circuit" with scale factor of 3.
+# Link to docs: https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.zne.scaling.folding.fold_global
+
+print("Globally folded GHZ circuit:")
+print(globally_folded_circuit)
+```
+
+In both cases, the results are longer circuits which are more sensitive to noise. Those circuits can be used to evaluate noise scaled expectation values.
+
+For example, let's use global folding to evaluate a list of noise scaled expectation values.
+
+```{code-cell} ipython3
+scale_factors = [1.0, 2.0, 3.0]
+# It is usually better apply unitary folding to the compiled circuit
+noise_scaled_circuits = [zne.scaling.fold_global(compiled_circuit, s) for s in scale_factors]
+
+# We run all the noise scaled circuits on the noisy backend
+noise_scaled_vals = [execute(c) for c in noise_scaled_circuits]
+
+print("Noise-scaled expectation values:", noise_scaled_vals)
+```
++++
+### STEP 2: Inference of the ideal result via zero-noise extrapolation
+Given the list of noise scaled expectation values, one can extrapolate the zero-noise limit. This is the final classical post-processing step.
+
+
+```{code-cell} ipython3
+# Initialize a Richardson extrapolation object
+richardson_factory = zne.RichardsonFactory(scale_factors)
+
+# Load the previously measured data
+for s, val in zip(scale_factors, noise_scaled_vals):
+    richardson_factory.push({"scale_factor": s}, val)
+
+print("The Richardson zero-noise extrapolation is:", richardson_factory.reduce())
+_ = richardson_factory.plot_fit()
+```
+
+```{code-cell} ipython3
+:tags: ["raises-exception"]
```

Comment:
Here and in similar other code cells, instead of using ` :tags: ["raises-exception"]` it is probably more appropriate to use `:tags: ["skip-execution"]`